src/Transition/OutputRulesFiles/OutputChanges9-3-0-to-9-4-0.md

```diff
@@ -90,3 +90,10 @@ Now subsurfaces are at the end of each group of zone surfaces.Many reports prese
 Changed outputs include the rdd, edd, eso (and resulting csv), shd, and sci output files.
 
 See [PR#7847](https://github.com/NREL/EnergyPlus/pull/7847)
+
+### Output:Variables, Output:Meter, Output:Meter:MeterFileOnly, Output:Table:Monthly
```

Comment:
And here, also mention table output headings moving from "Fuel Oil #1" to "Fuel Oil No 1" and 2.

Reply:
Done.